docs/Config_Reference.md

```diff
@@ -4410,6 +4410,62 @@ host_mcu:
 #   (True sets CFG5 high, False sets it low). The default is True.
 ```
 
+### [recore]
+
+The "recore" section is custom to recore boards.
+It adds support for Recore boards by resetting the over current protection
+(INA381) and enabling the high power section.
+
+Furthermore, there is a programmable gain amplifier for all temperature inputs
+allowing both thermistors, thermocouples and PT1000s to be used on all inputs as
+well as PT100 with an external amplifier.
+
+To use a thermistor, set the gain to 1, the pull-up to 1 and the offset to 0.
+
+To use a thermocouple set the gain to 100, the pull-up 0 and the offset to 1.
+```
+[recore]
+revision: A6
+#   Hardware revision. A6 and A7 are supported.
+# gain_t0: 1
+# gain_t1: 1
+# gain_t2: 1
+# gain_t3: 1
+#   Input section gain, values 1 or 100 are allowed.
+# pullup_t0: 1
+# pullup_t1: 1
+# pullup_t2: 1
+# pullup_t3: 1
+#   Programmable 4.7 K pull-up. 0 = off, 1 = on
+# offset_t0: 0
+# offset_t1: 0
+# offset_t2: 0
+# offset_t3: 0
+#   Programmable voltage offset on the input. 0 = off, 1 = on
```

Comment:
Please expand on the field comments here.  It's not clear to me what each field does.  It's also good to explicitly state what the default is (or to state the field is required).